samples/package.json

```diff
@@ -8,10 +8,11 @@
     "node": ">=8"
   },
   "scripts": {
-    "test": "mocha system-test"
+    "test": "mocha system-test system-test/v3beta1/"
   },
   "dependencies": {
     "@google-cloud/translate": "^3.0.0",
+    "@google-cloud/storage": "^2.4.3",
```

Comment:
It looks like this is only used in the tests.  Can we move this to `devDependencies`?